src/app/caesar-portal/common/services/group.service.ts

```diff
@@ -30,4 +69,13 @@ export class GroupService {
   private extractGroupsData(obj) {
     return new Group(obj.groupId, obj.name, obj.startDate, obj.finishDate, obj.experts);
   }
+
+  getGroup () {
+    return this.group;
+  }
+
+  setGroup (group: Group) {
+    this.group = Object.assign(this.group, group);
```

Comment:
if you need this group only inside of your component then keep it in component. If you need to organize communication between components, or some other components/services can influence this group use Subjects instead of just saving group inside of service, so you can notify everyone interested that group updated